initrd/detect.go

```diff
@@ -20,6 +20,8 @@ func New(ctx context.Context, path string, opts ...InitrdOption) (Initrd, error)
 		return builder, nil
 	} else if builder, err := NewFromOCIImage(ctx, path, opts...); err == nil {
 		return builder, nil
+	} else if builder, err := NewFromTarball(ctx, path, opts...); err == nil {
+		return builder, nil
```

Comment:
I this should sit somewhere between `NewFromFile` and `NewFromDockerfile` cause the tarball is a file, but a specific File, kind of like the Dockerfile